rca/people/utils.py

```diff
@@ -86,8 +86,8 @@ class StudentPagePromoteTab(ObjectList):
     # As a side effect: If all fields are hidden, the panel is hidden for Students
 
     class BoundPanel(ObjectList.BoundPanel):
-        def __init__(self, panel, instance, request, form):
-            super().__init__(panel, instance, request, form)
+        def __init__(self, *args, **kwargs):
+            super().__init__(*args, **kwargs)
```

Comment:
This seems to avoid the error:
```
Traceback (most recent call last):
  File "/app/rca/people/tests/test_utils.py", line 51, in test_create_page_with_per_request_custom_edit_handlers
    response = self.client.get(
…
File "/venv/lib/python3.8/site-packages/wagtail/admin/panels.py", line 259, in get_bound_panel
    return self.BoundPanel(
TypeError: __init__() got an unexpected keyword argument 'prefix'
```

Reply:
Agreed. [v3](https://github.com/wagtail/wagtail/blob/611b7a52dd549cf3d1befe32a0d59ea51f23d21b/wagtail/admin/panels.py#L292) - [v4](https://github.com/wagtail/wagtail/blob/d5ed0673f112c8f41731b14994146e27273d19e3/wagtail/admin/panels.py#L308)

Reply:
Here I tested this and following the Wagtail docs: https://docs.wagtail.org/en/stable/releases/3.0.html#api-changes-to-panels-edithandlers it can be just:

```
def __init__(self, **kwargs):
            super().__init__(**kwargs)
```
I tested it locally and it works OK that way.